index.html

```diff
@@ -34,14 +34,21 @@
 
       const ldclient = LDClient.initialize(clientSideID, user);
 
-      function render() {
-        const flagValue = ldclient.variation(flagKey, false);
-        const label = 'Feature flag ' + flagKey + ' is ' + flagValue + ' for this user';
-        div.replaceChild(document.createTextNode(label), div.firstChild);
-      }
+      ldclient.waitForInitialization().then(() => {
```

Comment:
I'm not sure if this is the correct way to do this... I'm attempting to follow instructions [in the reference docs](https://docs.launchdarkly.com/sdk/client-side/javascript#initializing-the-client) and [SDK docs](https://launchdarkly.github.io/js-client-sdk/interfaces/_launchdarkly_js_client_sdk_.ldclient.html#waitforinitialization).